lib/elixir/lib/kernel.ex

```diff
@@ -1585,9 +1585,13 @@ defmodule Kernel do
       true  -> opts.width
       false -> :infinity
     end
-    IO.iodata_to_binary(
```

Comment:
Let's revert those changes and instead of `:default`, we can call it `:reset`. This means we don't really support a default color but only the parameter to reset to. If someone wants an initial together, they could do the wrapping and properly set `:reset`.